### PR TITLE
Allow dropping a user not in the server

### PR DESCRIPTION
**Pull request by AlphaKretin**
## Description

Handle when their Member data doesn't exist
Relevant to #465, though doesn't close it entirely

## Checklist

- [x] I am following the [contributing guidelines](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/.github/CONTRIBUTING.md).
- [ ] I have updated any relevant [user guides](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/guides).
- [ ] I have updated any relevant [documentation](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/docs).
- [ ] I have updated the [changelog](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/changelog.md), or this change is not user-facing.

### CODE REVIEW

**Review by kevinlul — APPROVED** [review]

**Comment by AlphaKretin** [comment]
While testing, the schema edit caused issues on my end, so we're gonna have to figure that out when we merge

**Comment by kevinlul** [comment]
Is synchronize bad at mutating SQLite tables?

**Review by kevinlul — APPROVED** [review]

**Comment by kevinlul** [comment]
synchronize will be fine as long as there is no data in the deck table, due to the non-null constraint on the new column

File: src/slash/forcedrop.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {
 	chatInputApplicationCommandMention,
 	ChatInputCommandInteraction,
 	ContextMenuCommandBuilder,
+	GuildMember,
```
Comment (kevinlul) [inline src/slash/forcedrop.ts:8]: Should be good to go, but unused import

**Pull request merged**